src/api/app/models/project/update_from_xml_command.rb

```diff
@@ -28,7 +28,8 @@ def run(xmlhash, force = nil)
       project.remoteurl = xmlhash.value('remoteurl')
       project.remoteproject = xmlhash.value('remoteproject')
       project.scmsync = xmlhash.value('scmsync')
-      project.kind = xmlhash.value('kind') if xmlhash.value('kind').present?
+      project.kind = xmlhash.value('kind').presence || 'standard'
```

Comment:
I've searched backwards in the commit history about this change. The title of the pull request, the commit title and the description of the commit gave me a hint about fixing something, or reverting some change. I found these changes:

https://github.com/openSUSE/open-build-service/commit/c03f3e0ebbc2e97714ba684a71b8dd14a00e9578#diff-40f55459121e262fa08c43869c11eb281f30a3b5e0624fa8aee3f3e970e103f5L441-R451

Shouldn't we describe better why `'standard'` should be the default value for the `kind` of a project?